client/src/pages/Profile.js

```diff
@@ -0,0 +1,227 @@
+import React from "react";
+import { Popover, WhiteSpace } from "antd-mobile";
+import { MenuOutlined, EditOutlined } from "@ant-design/icons";
+const linkedinIcon = require("../assets/icons/social-linkedin.svg");
+const twitterIcon = require("../assets/icons/social-twitter.svg");
+const offerHelpInactive = require("../assets/help-gesture-unselected.svg");
+const needHelpInactive = require("../assets/thermometer-unselected.svg");
+
+// dummy data props,context, redux etc
+const firstName = "Cees";
+const lastName = "Wang";
+const about =
+  "  Lorem ipsum dolor sit amet, consectetuer adipiscing elit, sed diamnonummy nibh euismod tincidunt ut laoreet dolore magna";
+const email = "ceeswang@Test.com";
+const location = "NY, USA";
+const needHelp = true;
+const Item = Popover.Item;
+
+function getInitials(firstName, lastName) {
```

Comment:
could be nice to move to utils or some shared file in profile components if used more than once